WindowsServerDocs/troubleshoot/windows-server-support-solutions.md

```diff
@@ -1,183 +1,157 @@
-                        <a href="../identity/ad-fs/deployment/upgrading-to-ad-fs-in-windows-server-2016.md">Upgrading to AD FS in Windows Server 2016 using a WID database</a></p>
-<hr>
-                        <p><a href="../identity/ad-fs/operations/configure-device-based-conditional-access-on-premises.md">Windows 10 Sign on – enabling device authentication with AD FS</a></p>
-<hr>
-                        <p><a href="../identity/ad-fs/operations/manage-ssl-certificates-ad-fs-wap-2016.md">Managing SSL Certificates in AD FS and WAP in Windows Server 2016</a></p>
-<hr>
-                        <p><a href="../identity/ad-fs/operations/access-control-policies-in-ad-fs.md">Access Control Policies in Windows Server 2016 AD FS</a></p>
-                      </div>
-                 </div>
-            </div>
-        </div>
-    </li>
-<li>
- <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="../media/i-troubleshoot.svg" alt="" />
-                        </div>
-                    </div>
-                    <div class="cardText">
-                        <h3>Active Directory replication</h3>
-<hr> 
-                         <a href="../identity/ad-ds/manage/troubleshoot/troubleshooting-active-directory-replication-problems.md">Troubleshooting Active Directory Replication Problems</a>
-<hr>
-                         <a href="https://www.microsoft.com/en-in/download/details.aspx?id=30005">e2e: How to troubleshoot common Active Directory replication errors</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/3108513">Download Active Directory Replication Status Tool from the Microsoft Download Center</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/2028495">Troubleshooting AD Replication error 8606: Insufficient attributes were given to create an object</a></p>
-<hr>
-                         <a href="https://support.microsoft.com/kb/837932">Event ID 2108 and Event ID 1084 occur during inbound replication of Active Directory in Windows 2000 Server and in Windows Server 2003</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/2645996">Troubleshooting AD Replication error 8451: The replication operation encountered a database error</a>
-<hr>
-                         <a href="https://support.microsoft.com/kb/2025726">Troubleshooting AD Replication error 1127: While accessing the hard disk, a disk operation failed even after retries</a>
-<hr>
-                         <a href="https://technet.microsoft.com/library/cc816907.aspx">Clean Up Server Metadata</a>
-                    </div>
-                </div>
-            </div>
-        </div>
-    </li>
-</ul>
\ No newline at end of file
+# Troubleshooting Windows Server components
+
+>- [!TIP]
```

Comment:
Not sure if you want to add a bullet point for a Tip note. The HTML version of the page will automatically add a minor graphic light bulb in front of the word "**Tip**" , so an extra bullet point could look a bit redundant. See the top of the page https://docs.microsoft.com/en-us/windows-server/troubleshoot/windows-server-support-solutions for how a default Tip note looks in the HTML version.